config/database.php

```diff
@@ -40,7 +40,7 @@
             'url' => env('DATABASE_URL'),
             'database' => env('DB_DATABASE', database_path('database.sqlite')),
             'prefix' => '',
-            'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
+            'foreign_key_constraints' => true,
```

Comment:
I think this should not be changed here, edit DB_FOREIGN_KEYS constant in .env file instead.

Reply:
I would enforce it in the production environment to ensure that foreign key constraints are always applied. What could be the benefits of keeping this option in the .env file?

Reply:
I do not think we would like to use sqlite in production :) With this change, you can not even turn it off for your local developer environment without code changes. But I do not insist on this.

Reply:
Hm, fine, we can leave it more flexible.